src/main/java/mods/railcraft/common/plugins/forge/LootPlugin.java

```diff
@@ -70,7 +70,7 @@ public static void addLoot(RailcraftItem item, IItemMetaEnum meta, int minStack,
 
     public static void addLootUnique(RailcraftItem item, IItemMetaEnum meta, int minStack, int maxStack, Type type) {
         ItemStack stack = item.getStack(meta);
-        addLoot(stack, minStack, maxStack, stack.getUnlocalizedName(), type.locations);
+        addLoot(stack, minStack, maxStack, item.getBaseTag(), type.locations);
```

Comment:
I wrote this function the other day specifically because item.getBaseTag() is too generic.